src/lib.rs

```diff
@@ -9,7 +9,7 @@
         missing_debug_implementations, missing_copy_implementations,
         unsafe_code,
         unstable_features,
-        unused_import_braces, unused_qualifications)]
```

Comment:
The linter around `unused_qualitifications` is buggy. When I added the numeric constraints to `AddAssign for CKMS<T>` – which are necessary to compile – this linter flagged them as unused.